packages/squiggle-lang/src/public/SqValue/SqPlot.ts

```diff
@@ -280,10 +284,51 @@ export class SqRelativeValuesPlot extends SqAbstractPlot<"relativeValues"> {
     );
   }
 }
+export class SqTable extends SqAbstractPlot<"table"> {
+  tag = "table" as const;
+
+  get elements(): SqValue[] {
+    return this._value.elements.map((r) => wrapValue(r, this.context));
+  }
+
+  get fns(): SqLambda[] {
+    return this._value.fns.map(
+      (fn) =>
+        new SqLambda(fn, this.context ? this.context.extend("fn") : undefined)
+    );
+  }
+
+  item(row: number, column: number, env: Env): Result.result<SqValue, SqError> {
+    const response = this.fns[column].call([this.elements[row]], env);
+    const context: SqValueContext | undefined =
+      this.context && this.context.extend(`item(${row}, ${column})`);
```

Comment:
I really wasn't sure about this, curious to get help here.

Reply:
My guess is that in for this PR, assuming it's not too much work, we get ``get()`` to work, and call it a day. I think that making cells focusable in a generic way would require some real UI thinking. 

We could later have a way for making key elements that might want to be expanded, focusable.